lib/gui/pages/settings/controllers/settings.js

```diff
@@ -86,6 +86,14 @@ module.exports = function(WarningModalService, SettingsModel, ErrorService, Anal
       dangerous
     });
 
+    if (setting === 'errorReporting') {
```

Comment:
I feel we shouldn't put logic on how to react to certain setting here, and that we should query the redux store for the value instead. Maybe we can add another `.run()` block at `app.js` that subscribes to the store and checks the setting there? e.g: like https://github.com/resin-io/etcher/blob/master/lib/gui/app.js#L132